tests/phpunit/tests/post.php

```diff
@@ -1844,4 +1844,53 @@ public function test_unstick_post_with_non_sticky_post_id_does_not_update_option
 		unstick_post( 3 );
 		$this->assertSameSets( array( 1, 2, 2 ), get_option( 'sticky_posts' ) );
 	}
+
+	/**
+	 * Ensure updated post have a different slug then already publish post with same name.
+	 *
+	 * @ticket 50447
+	 * @covers ::wp_update_post
+	 */
+	public function test_updated_draft_post_have_unique_slug_from_publish_post_with_same_title() {
+
+		// Publish post metainfo.
+		$post = array(
+			'post_name'   => 'test',
+			'post_title'  => 'test',
+			'post_status' => 'publish',
+		);
+
+		// Add post.
+		$post_id = wp_insert_post( $post );
+
+		// Testcase for publish post with $post_id
+		$this->assertSame( 102, $post_id );
```

Comment:
How do you know that 102 will be post ID whenever test run? I believe no need to assert for post ID.